### ISSUE TITLE
GtfsRtVehiclePositionsReaderBase.java seems to expect gtfs vehicle position timestamps  to be in milliseconds rather than seconds.

### ISSUE THREAD

**Issue by scrudden** [Transitime/core#12]
The GTFS-rt spec states the time stamps for vehicle positions are to be in seconds from epoch not milliseconds.

```
            if (vehicle.hasTimestamp())
                gpsTime = vehicle.getTimestamp();
            else
                gpsTime = System.currentTimeMillis();
```

I expect this should be 

```
        if (vehicle.hasTimestamp())
                gpsTime = vehicle.getTimestamp()*1000;
            else
                gpsTime = System.currentTimeMillis();
```

**Comment by skibu** [comment]
You are correct. Glad you found it. gpsTime should be in msec yet vehicle.getTimestamp() returns time in seconds. 

By the way, I prefer using Time.MS_PER_SEC or Time.SEC_IN_MSECS instead of 1000 because it makes it more clear why doing the conversion.

Let me know if you will create a pull request or if I should make the change myself.

**Comment by barbeau** [comment]
I'd suggest using Java TimeUnit.  IMHO, it makes it very clear what
conversion is happening and is native Java.  For usage examples see:
http://www.javaworld.com/article/2074114/core-java/the-highly-useful-java-timeunit-enum.html

**Comment by skibu** [comment]
Sean, thanks for the tip on TimeUnit. I've been using constants for so long I didn't realize TimeUnit was a viable alternative.

**Comment by skibu** [comment]
Closed due to pull request #16.
